core/src/config/secrets.ts

```diff
@@ -17,31 +17,32 @@ import difference from "lodash-es/difference.js"
 import { ConfigurationError } from "../exceptions.js"
 import { CONTEXT_RESOLVE_KEY_NOT_FOUND } from "../template/ast.js"
 
-/**
- * Gathers secret references in configs and throws an error if one or more referenced secrets isn't present (or has
- * blank values) in the provided secrets map.
- *
- * Prefix should be e.g. "Module" or "Provider" (used when generating error messages).
- */
-export function throwOnMissingSecretKeys(
-  configs: ObjectWithName[],
-  context: ConfigContext,
-  secrets: StringMap,
-  prefix: string,
-  log?: Log
-) {
-  const allMissing: [string, ContextKeySegment[]][] = [] // [[key, missing keys]]
-  for (const config of configs) {
-    const missing = detectMissingSecretKeys(config, context, secrets)
-    if (missing.length > 0) {
-      allMissing.push([config.name, missing])
-    }
+function getMessageFooter({ loadedKeys, isLoggedIn }: { loadedKeys: string[]; isLoggedIn: boolean }) {
+  if (!isLoggedIn) {
+    return "You are not logged in. Log in to get access to Secrets in Garden Cloud."
   }
 
-  if (allMissing.length === 0) {
-    return
+  if (loadedKeys.length === 0) {
+    return deline`
+      Note: No secrets have been loaded. If you have defined secrets for the current project and environment in Garden
+      Cloud, this may indicate a problem with your configuration.
```

Comment:
This is still extremely confusing. I'd suggest the following, to improve the user experience:
```suggestion
      Note: You can manage secrets in Garden Cloud. No secrets have been defined for the current project and environment. See also https://cloud.docs.garden.io/features/secrets
```

Reply:
Sure, updated this in 5d676e36db6d465cac52397f171453f1849c09be.